### ISSUE TITLE
read raw image

### ISSUE THREAD

**Issue by fforoutan** [gravi-toni/subpixel-edges#2]
Hello sir,

How can I give a raw photo to this algorithm? Everything I try gives an error

**Comment by readmees** [comment]
yes just read the photo as a numpy array with skimage.io.imread and use that as input for the algorithm
